src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/GivenAResolveRuntimePackAssetsTask.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using FluentAssertions;
+using Microsoft.Build.Utilities;
+using Xunit;
+
+namespace Microsoft.NET.Build.Tasks.UnitTests
+{
+    public class GivenAResolveRuntimePackAssetsTask
+    {
+        [Fact]
+        public void ItFiltersSatelliteResources()
+        {
+            string projectDirectory = Path.Combine(Path.GetTempPath(), "runtimePackSatelliteAssets");
```

Comment:
nit: can you use _testAssetsManager.CreateTestDirectory() instead? That way you won't have to delete it at the end.